src/main/resources/locales/en/subscription-configure-list-confirmed.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Subscriptions updated",
+  "header": "Subscriptions updated",
+  "subheader": "Your subscription(s) have been updated successfully",
+  "p1": "To continue, you can go to ",
+  "link_text": "your account",
+  "p2": " in order to:",
+  "addEmailSubscription": "add a new email subscription",
+  "manageYourCurrentSubscriptions": "manage your current email subscriptions",
+  "manageYourListTypeSubscriptions": "Further refine the list types per court received",
```

Comment:
Just wondering if this should be a capital or not?

The mock up does show it but why are the rest on the page lower case.

Seems like we either need all with capital or none?

Reply:
As per discussion with Chris C, we do not need to include third bullet point. So, I have removed and and updated the rest of content changes on the page.